test/game.test.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect = require('chai').expect;
+var Grid = require('../app/scripts/grid');
+var Game = require('../app/scripts/game');
+
+describe('Game', function () {
+
+  it('should determine a winner', function () {
+    var game = new Game({
+      grid: new Grid({
+        columnCount: 7,
+        rowCount: 6
+      })
+    });
+    game.startGame();
+    game.placePendingChip({column: 0}); // P1
+    game.endTurn();
+
+    // Winners haven't won until they've won.
+    game.checkForWin();
```

Comment:
Since 6b673a4e256bd43636fed5a3c62c47644c555f0f, `placePendingChip` automatically calls `checkForWin`, so there is no need to call it explicitly here or anywhere below.